recipes/spdlog/all/conanfile.py

```diff
@@ -82,6 +84,9 @@ def _disable_werror(self):
         tools.replace_in_file(os.path.join(self._source_subfolder, "cmake", "utils.cmake"), "/WX", "")
 
     def build(self):
+        if tools.Version(self.version) < "1.7" and \
+           tools.Version(self.deps_cpp_info["fmt"].version) >= 7:
+            raise ConanInvalidConfiguration("The project spdlog {} requires fmt <7.x".format(self.version))
```

Comment:
Since there's API/ABI breaks in fmt, should this check be added for the other versions as well?

Reply:
Yes, if you know the version. Please, copy this valid to your PR, so we can fix #2228